packages/viewer/src/stream/result.ts

```diff
@@ -0,0 +1,5 @@
+import { StreamApi } from '@vertexvis/stream-api';
+
+export class Result<T = undefined> {
+  public constructor(public data: T, private stream: StreamApi) {}
+}
```

Comment:
I'd maybe make `stream` be part of subclasses to keep this more generic. Could also probably turn this into an interface.